dojo_toolkit/test_runner.py

```diff
@@ -0,0 +1,14 @@
+from subprocess import Popen
+
+
+class BaseTestRunner():
+    def __init__(self, code_path):
+        self.code_path = code_path
+
+
+class DoctestTestRunner(BaseTestRunner):
+    def run(self):
+        cmd = "python -m doctest {}/*.py".format(self.code_path)
```

Comment:
Before showing the things, for minimal confusion is always good to clean the terminal and show **only** the new errors or test pass message